vendor/k8s.io/kubernetes/test/e2e/scheduling/preemption.go

```diff
@@ -242,17 +303,18 @@ var _ = SIGDescribe("SchedulerPreemption [Serial]", func() {
 			node := nodeList.Items[i]
 			currentCpuUsage, currentMemUsage := getCurrentPodUsageOnTheNode(node.Name, allPods.Items, podRequestedResource)
 			framework.Logf("Current cpu usage and memory usage is %v, %v", currentCpuUsage, currentMemUsage)
-			currentNode, err := cs.CoreV1().Nodes().Get(node.Name, metav1.GetOptions{})
-			framework.ExpectNoError(err)
-			cpuAllocatable, found := currentNode.Status.Allocatable["cpu"]
+			cpuAllocatable, found := node.Status.Allocatable["cpu"]
 			Expect(found).To(Equal(true))
 			milliCPU := cpuAllocatable.MilliValue()
-			milliCPU = milliCPU * 10 / 100
-			memAllocatable, found := currentNode.Status.Allocatable["memory"]
+			// Just to be tolerant use 0.6 of resources available on the node
+			milliCPU = int64(float64(milliCPU-currentMemUsage) * float64(0.6))
```

Comment:
@damemi it looks like this ends up being a negative number in the tests: `Pod \"pod0-sched-preemption-medium-priority\" is invalid: spec.containers[0].resources.requests[cpu]: Invalid value: \"-912259020m\": must be greater than or equal to 0` maybe you'd need similar checks for utilization like in the other test?